core/src/main/resources/schema/app/20211119143913_DISTX-683_add_secrets_column_to_customconfigurations_properties_table.sql

```diff
@@ -0,0 +1,13 @@
+-- // DISTX-683 add secrets column to customconfigurations_properties table
+-- Migration SQL that makes the change goes here.
+ALTER TABLE customconfigurations_properties ADD COLUMN IF NOT EXISTS secretvalue text NOT NULL DEFAULT '{}';
```

Comment:
as we are still using postgres 9.x please split this into separate step like it's described here: https://dba.stackexchange.com/a/216163